packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -19,7 +19,7 @@ export const SharedPostCardFooter = ({
   return (
     <div
       className={classNames(
-        'p-3 gap-3 mb-2 rounded-12 border border-theme-divider-tertiary flex w-fit flex-auto h-auto min-h-0',
+        'p-3 gap-3 mb-2 rounded-12 border border-theme-divider-tertiary flex w-fit flex-auto h-auto min-h-0 w-full',
```

Comment:
There are contradicting widths now.
In our tailwind version only 1 will win and think it's currently based on alphabetical order, maybe double check which one get's rendered and delete the other?

Refering to `w-fit` and `w-full`

Reply:
That's what's a bit annoying with using `classNames` function, the linter doesn't catch it 😬
That's why I (try to) always paste the classes in a regular `className` to get the linter to do its thing.

Reply:
Removed w-fit, that was my intention, just missed removing it 😓